library/Messages/tr_TR.php

```diff
@@ -0,0 +1,954 @@
+<?php
+namespace Respect\Validation;
+
+use Respect\Validation\Exceptions\ValidationException;
+use Respect\Validation\Exceptions\AbstractGroupedException;
+use Respect\Validation\Exceptions\AlphaException;
+use Respect\Validation\Exceptions\AlwaysInvalidException;
+use Respect\Validation\Exceptions\AttributeException;
+use Respect\Validation\Exceptions\BetweenException;
+use Respect\Validation\Exceptions\DateException;
+use Respect\Validation\Exceptions\EqualsException;
+use Respect\Validation\Exceptions\IpException;
+use Respect\Validation\Exceptions\LengthException;
+use Respect\Validation\Exceptions\MaxException;
+use Respect\Validation\Exceptions\MinException;
+use Respect\Validation\Exceptions\NotEmptyException;
+
+$messages = array(
```

Comment:
Probably you're right. On the other hand, I think usage should still be like this:

```php
v::setLocale('en_US');

try {
    v::int()->check("hello");

} catch ($exception) {
    echo $exception->getMainMessage(); // automatically localized message
}
```